packages/sp/src/lists.ts

```diff
@@ -435,6 +435,36 @@ export class List extends SharePointQueryableSecurable {
     public getListItemEntityTypeFullName(): Promise<string> {
         return this.clone(List, null, false).select("ListItemEntityTypeFullName").get<{ ListItemEntityTypeFullName: string }>().then(o => o.ListItemEntityTypeFullName);
     }
+
+    /**
+     * Creates an item using path (in a folder), validates and sets its field values.
+     *
+     * @param formValues The fields to change and their new values.
+     * @param decodedUrl Path decoded url; folder's server relative path.
+     * @param bNewDocumentUpdate true if the list item is a document being updated after upload; otherwise false.
+     * @param checkInComment Optional check in comment.
+     */
+    public addValidateUpdateItemUsingPath(
+        formValues: ListItemFormUpdateValue[],
+        decodedUrl: string,
+        bNewDocumentUpdate = false,
+        checkInComment?: string,
+    ): Promise<ListItemFormUpdateValue[]> {
+        return this.clone(List, "AddValidateUpdateItemUsingPath()").postCore({
+            body: JSON.stringify({
+                bNewDocumentUpdate,
+                checkInComment,
+                formValues,
+                listItemCreateInfo: {
+                    FolderPath: {
+                        DecodedUrl: decodedUrl,
+                        __metadata: { type: "SP.ResourcePath" },
+                    },
+                    __metadata: { type: "SP.ListItemCreationInformationUsingPath" },
+                },
+            }),
+        }).then(res => res.AddValidateUpdateItemUsingPath.results);
```

Comment:
Have you tried this in both verbose and non-verbose mode? Usually the method name doesn't come back in minimal.

Reply:
You're right. Only checked in a verbose mode, my bad. Just fixed that.